src/app/components/home/home.component.ts

```diff
@@ -578,7 +578,7 @@ export class HomeComponent implements OnInit, AfterViewInit {
       if (ev.dataTransfer.files.length > 0) {
         const fullPath = ev.dataTransfer.files[0].path;
         ev.preventDefault();
-        if (fullPath.slice(-4) === '.vha') {
+        if (fullPath.slice(-5) === '.vha2') {
```

Comment:
I feel like we should have some global `FILE_EXTENSION='.vha2'` for everywhere that uses the extension, and then also should this be `fullPath.endsWith(FILE_EXTENSION)`?

Reply:
Thanks for the `.endsWith()` method -- I've not come across it before 😰 it's great to know it's available 😍 

Having a global would make search-and-replace unnecessary if we ever upgrade to `.vha3` :trollface: 

Unsure what other benefit we'll get by having a constant rather than a string ... omitting spelling mistakes is the only one that comes to mind now 🤔

Reply:
Well, I count 6 `.vha2` occurrences in this change - I'd do it for anything over 2! 😄 

Maybe a better name like `globals.ext` would be prettier! 👍